### PR TITLE
Added stylesheet to index.css

### PR DESCRIPTION
**Pull request by jowen03**
Fixed issue before so it's now added correctly

### CODE REVIEW

File: frontend/src/index.css [hunk 1]
```diff
@@ -1,12 +1,59 @@
...
+	font-size: : "48px";
+	font-family: : 'Open Sans';
+	font-weight: bold;
+
+}
+
+h2 {
+	color: #FFFFFF;
+	font-size: "36px";
+	font-family: 'Open Sans';
```
Comment (baueraidan) [inline frontend/src/index.css]: add fallback

File: frontend/src/index.css [hunk 2]
```diff
@@ -1,12 +1,59 @@
...
+	color: #FFFFFF;
+	font-size: : "48px";
+	font-family: : 'Open Sans';
+	font-weight: bold;
+
+}
+
+h2 {
+	color: #FFFFFF;
+	font-size: "36px";
```
Comment (baueraidan) [inline frontend/src/index.css]: no quotes

File: frontend/src/index.css [hunk 3]
```diff
@@ -1,12 +1,59 @@
...
+
+main {
+	backgroundColor: #FFFFFF;
+
+}
+
+h1 {
+	color: #FFFFFF;
+	font-size: : "48px";
+	font-family: : 'Open Sans';
```
Comment (baueraidan) [inline frontend/src/index.css]: fallback

File: frontend/src/index.css [hunk 4]
```diff
@@ -1,12 +1,59 @@
-
-html, body {
+html {
 	height: 100%;
 }
+
 body {
 	margin: 0;
 	padding: 0;
 	font-family: sans-serif;
```
Comment (baueraidan) [inline frontend/src/index.css]: this line can be removed because it's over-written on the next line

**Comment by bkcapretta** [comment]
Make sure to fix the edits Aidan mentioned!

**Pull request merged**